From 167768902ef2b12819120df582e7a59421329f38 Mon Sep 17 00:00:00 2001
From: Rajees P P <rajerpp1@in.ibm.com>
Date: Sat, 6 Jun 2026 20:12:58 +0530
Subject: [PATCH] Adding new ppe42 mcu type

---
 include/llvm/TargetParser/PPCTargetParser.def | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/llvm/TargetParser/PPCTargetParser.def b/include/llvm/TargetParser/PPCTargetParser.def
index da4be3e..53d1189 100644
--- a/include/llvm/TargetParser/PPCTargetParser.def
+++ b/include/llvm/TargetParser/PPCTargetParser.def
@@ -94,6 +94,7 @@ PPC_CPU("440",SYS_CALL,42,BUILTIN_PPC_FALSE,0)
 PPC_CPU("440fp",SYS_CALL,42,BUILTIN_PPC_FALSE,0)
 PPC_CPU("ppc440",SYS_CALL,42,BUILTIN_PPC_FALSE,0)
 PPC_CPU("450",BUILTIN_PPC_UNSUPPORTED,0,BUILTIN_PPC_UNSUPPORTED,0)
+PPC_CPU("ppe42",BUILTIN_PPC_UNSUPPORTED,0,BUILTIN_PPC_UNSUPPORTED,0)
 PPC_CPU("601",BUILTIN_PPC_UNSUPPORTED,0,BUILTIN_PPC_UNSUPPORTED,0)
 PPC_CPU("602",BUILTIN_PPC_UNSUPPORTED,0,BUILTIN_PPC_UNSUPPORTED,0)
 PPC_CPU("603",BUILTIN_PPC_UNSUPPORTED,0,BUILTIN_PPC_UNSUPPORTED,0)
-- 
2.48.1

